app/server/src/routes/enrollment/enrollment.route.js

```diff
@@ -8,16 +8,16 @@ const enrollmentRouter = express.Router();
 
 enrollmentRouter.post(
   "/",
-  // validate("createEnrollment"),
-  // handleValidation,
+  validate("createEnrollment"),
+  handleValidation,
   authorization,
   EnrollmentController.createEnrollment
 );
 
 enrollmentRouter.get(
   "/getEnrolledCourses",
-  //validate("getEnrolledCourses"),
-  //handleValidation,
+  validate("getEnrolledCourses"),
+  handleValidation,
```

Comment:
validation on get requests might be a problem

Reply:
removed validation!